contents/blog/best-pendo-alternatives.mdx

```diff
@@ -0,0 +1,465 @@
+
+2. **Whatfix:** A digital adoption and product analytics platform for enterprises providing onboarding, guidance, and training. 
+
+3. **Hotjar:** Connects heatmaps, recordings, and feedback to generate behavioral insights on users.
+
+4. **WalkMe:** A digital adoption tool focused on helping track, automate, and optimize the software use of large teams internally.
+
+5. **Heap:** A digital insights platform for understanding the end-to-end journey including analytics, autocapture, and session replay.
+
+6. **Appcues:** A focused product onboarding and surveying tool for boosting product adoption.
+
+7. **Smartlook:** Comprehensive user insight platform with analytics, session replay, heatmaps, and crash reports.
+
+This guide covers:
+
+- How these tools compare to Pendo.
+- Which key Pendo features they support.
+- What kind of users and companies use them, and why.
+
+## 1. PostHog
+
+- **Founded:** 2020
+- **Most similar to:** Heap, Pendo, Smartlook
+- **Typical users:** Engineers, product, and growth teams
+- **Typical customers:** Mid-size product-focused B2Bs and startups
+
+![PostHog](../images/blog/best-pendo-alternatives/posthog.png)
+
+### What is PostHog?
+
+[PostHog](/) (that's us 👋) is an open-source platform combining product analytics, session replay, user surveys, feature flags, and A/B testing into one product. This means it's not just an alternative to Pendo, but also tools like [LaunchDarkly](/blog/best-launchdarkly-alternatives) and [Amplitude](/blog/best-amplitude-alternatives).
+
+Typical PostHog users are engineers, founders, and product managers at startups and mid-size companies, particularly B2B companies. Customers include [AssemblyAI](/customers/assemblyai), [Hasura](/customers/hasura), [Vendasta](/customers/vendasta), and Airbus.
+
+### Key features
+
+- 📈 **Product analytics:** Funnels, user paths, retention analysis, custom trends, and dynamic user cohorts. Also supports [SQL insights](/docs/product-analytics/sql) for power users.
+
+- 💬 **Surveys:** Target surveys by event or user properties. Templates for [net promoter score (NPS)](/templates/nps-survey), [product-market fit (PMF)](/templates/pmf-survey) surveys, and more.
+
+- 📺 **Session replays:** Including event timelines, console logs, network activity, and 90-day data retention.
+
+- 🧪 **A/B tests:** Up to 9 test variations, primary and secondary metrics. Automatically calculate test duration, sample size, and statistical significance.
+
+- 🚩 **Feature flags:** Rollout features safely with local evaluation (for faster performance) and JSON payloads.
+
+### How does PostHog compare to Pendo?
+
+Both PostHog and Pendo are multi-product companies covering many use cases. Pendo focuses on in-app guides and internal tools. PostHog focuses on tools engineering and growth teams need like feature flags and A/B testing.
```

Comment:
I would emphasize product teams, since the people considering Pendo are likely PMs
```suggestion
Both PostHog and Pendo are multi-product companies covering many use cases. Pendo focuses on in-app guides and internal tools. PostHog focuses on tools product and engineering teams need like feature flags and A/B testing.
```